docs/content/modules/player-lifecycle-feature-module.md

```diff
@@ -14,49 +15,53 @@ Before reading this document, make sure you are familiar with:
 
 The Player lifecycle module is a feature module providing you with a simple player spawning and player lifecycle management implementation.
 
-### How to enable this module
+## How to enable this module
 
 To enable this module, you need to add the necessary systems to your workers in the `HandleWorkerConnectionEstablished()` of your [`WorkerConnector`]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector).
 After your worker has been created, you need to use the following code snippet:
 
 1. On a [client-worker]({{urlRoot}}/content/glossary#client-worker): `PlayerLifecycleHelper.AddClientSystems(Worker.World)`
+  * By default, the module automatically sends a player creation request as soon as the client worker connects to SpatialOS. To manually initiate player creation or provide arbitrary data in a player creation request, add an extra `false` argument when adding client systems: `PlayerLifecycleHelper.AddClientSystems(Worker.World, false);`
 1. On a [server-worker]({{urlRoot}}/content/glossary#server-worker): `PlayerLifecycleHelper.AddServerSystems(Worker.World)`
 
-### How to spawn a player entity
+## How to spawn a player entity
 
 To spawn a [SpatialOS entity]({{urlRoot}}/content/glossary#spatialos-entity) representing
 a player, you need to:
 
 1. Define the entity template for your player entity
 1. Specify an entity template for player creation
+1. (Optional) Manually request player creation
+
+>**NOTE: To pass in arbitrary serialized data with the player creation request, `AutoRequestPlayerCreation` must be set to false.**
```

Comment:
```suggestion
>**NOTE:** To pass in arbitrary serialized data with the player creation request, set `AutoRequestPlayerCreation` to false.
```
Assume that it's something our user has to do?  If not and it's just a check, "make sure set `AutoRequestPlayerCreation` is set to false."